### PR TITLE
FW-5285 Add Photo Gallery Read/Write APIs

### PR DESCRIPTION
**Pull request by bronson-c**
### Description of Changes
- Adds photo gallery read/write APIs with standard CRUD + patch functionality
- Removes photo galleries from the admin site

### Checklist
- [x] README / documentation has been updated if needed
- [x] PR title / commit messages contain Jira ticket number if applicable
- [x] Tests have been updated / created if applicable
- [x] Admin Panel has been updated if models have been added or modified
- [x] Migrations have been updated and committed if applicable
- [x] Team Postman workspace has been updated if applicable

### Reviewers Should Do The Following:
- [x] Review the code changes here
- [ ] Pull the branch and test locally

### Additional Notes
N/A

### CODE REVIEW

File: firstvoices/backend/views/gallery_views.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        return (
+            Gallery.objects.filter(site__slug=site[0].slug)
+            .prefetch_related(
+                "site",
+                "site__language",
+                "created_by",
+                "last_modified_by",
+                "galleryitem_set",
+            )
+            .select_related("cover_image")
```
Comment (NolanVH) [inline firstvoices/backend/views/gallery_views.py:115]: Some additional prefetching of the image and thumbnail fields could be added here to ensure low response times. Feel free to send me a message if you want to connect on this.

**Review by NolanVH — CHANGES_REQUESTED** [review]
Aside from the minor prefetching comment, these changes are looking great! Tested locally and all APIs are working as intended.

I am wondering if the gallery cover images are supposed to be unique across galleries or are multiple galleries supposed to be allowed to have the same cover image? Right now the cover image is a one-to-one field with a gallery and we may want to modify it to allow different galleries to use the same cover image.

**Review by sahil97 — APPROVED** [review]
Agree with Nolan's comment related to adding support for multiple galleries to use common image for cover.
except for that LGTM !

**Review by NolanVH — APPROVED** [review]
LGTM!

**Pull request merged**